### PR TITLE
Create sensitivePersonalInformationType.md

### PR DESCRIPTION
**Pull request by rgopikrishnan91**
Hi @kestewart and @bennetkl please review

### CODE REVIEW

**Review by kestewart — CHANGES_REQUESTED** [review]
name: sensors ??   doesn't look quite right.

File: model/Dataset/Properties/senstivePersonalInformationType.md
Comment (armintaenzertng) [inline model/Dataset/Properties/senstivePersonalInformationType.md:1]: The file name (and PR name) has a typo.

**Comment by kestewart** [comment]
@rgopikrishnan91 - can you fix?

**Comment by goneall** [comment]
Superseded by PR #184 which includes feedback from @armintaenzertng

**Review by bennetkl — APPROVED** [review]
Please fix typo in file name

**Pull request closed without merging**